editions/test/tiddlers/tests/test-attachment.js

```diff
@@ -0,0 +1,168 @@
+/*\
+title: test-attachment.js
+type: application/javascript
+tags: [[$:/tags/test-spec]]
```

Comment:
These tests should reside within the multiwikiserver plugin, and the tiddlywiki.info for the `editions/test` edition should be modified to include the multiwikiserver plugin. (This is the approach that the geospatial plugin uses for its tests).